server/appengine/src/main/java/who/WhoServletModule.java

```diff
@@ -25,6 +25,19 @@
     serve("/terms").with(forwardTo("/terms.pdf"));
     serve("/privacy").with(forwardTo("/privacy.pdf"));
 
+    // TODO: Make this much better.
+    final String[] bundles = new String[]{"news_index", "get_the_facts", "home_index", "learn_index", "protect_yourself", "symptom_checker", "travel_advice", "your_questions_answered"};
+    for (final String bundle : bundles) 
+    {
+      for (int i = 0; i < 26; i++) {
```

Comment:
This is fine for now, but thinking through how this changes in the future... do you have a plan for how this would update if a country ends up requiring a different content bundle? E.g. exclusion list of paths, reading from the actual files, etc?

Reply:
Config file seems best, but let's discuss and design.  Just doing this to launch for now.